engine/src/main/resources/templates/FleetManagement/imageCreationBody.html

```diff
@@ -0,0 +1,50 @@
+{#include Edge/insightsEmailBody}
+{#content-title}Egde Management Image Notification - {action.timestamp.toStringFormat()}{/content-title}
```

Comment:
Should `Egde` be replaced with `Fleet Management`? If not, the typo will need to be fixed: `Egde` -> `Edge`.